src/infrastructure/healthchecks/healthchecks.spec.ts

```diff
@@ -3,14 +3,17 @@ import { asFunction } from 'awilix'
 import type { FastifyInstance } from 'fastify'
 import type Redis from 'ioredis'
 import { afterEach, beforeEach, describe, expect, it } from 'vitest'
-import type { AppInstance } from '../app.js'
-import { getApp } from '../app.js'
+import type { AppInstance } from '../../app.js'
+import { getApp } from '../../app.js'
 
-import { dbHealthCheck, redisHealthCheck } from './healthchecks.js'
+import { resetHealthcheckStores } from './healthchecks.js'
+import { dbHealthCheck, redisHealthCheck } from './healthchecksWrappers.js'
 
 const createRedisMock = (pingLatency: number, response = 'PONG') =>
   ({
     ping: () => new Promise((resolve) => setTimeout(resolve, pingLatency, response)),
+    // set: () => Promise.resolve(),
+    // get: () => Promise.resolve('dummy')
```

Comment:
🟢 Is this commented out on purpose to be later on reverted or can it be deleted?